app/pages/Logs.js

```diff
@@ -7,6 +7,23 @@ const remote = require('@electron/remote');
 
 const config = remote.getGlobal('config');
 
+const STATUS_COLOR_MAP = {
+  success: 'green',
+  info: 'blue',
+  warning: 'yellow',
+  error: 'red',
+  debug: 'darkgrey',
+};
+const STATUS_ICON_MAP = {
+  success: 'check',
+  info: 'info circle',
+  warning: 'warning sign',
+  error: 'window close',
```

Comment:
```suggestion
  error: 'x',
```
Should actually be `x`; `window close`, `close`, and `delete` all makes the button clickable with weird margins.